### PR TITLE
Active admin

### PR DESCRIPTION
**Pull request by santyalcala**
#Summary

- Active admin gem added for index, show, delete and create books, users and rents.

# Trello card

- https://trello.com/c/5FiK9Lhe/10-instalar-y-configurar-active-admin

# Screenshot

- 
active admin

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (Dinuuu) [inline app/admin/dashboard.rb]: delete the commented code

File: app/admin/rent.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      f.input :from, as: :datetime_picker
+      f.input :to, as: :datetime_picker
+    end
+    f.actions
+  end
+
+  #
+  # or
+  #
+  # permit_params do
```
Comment (Dinuuu) [inline app/admin/rent.rb]: idem

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< 1a6d592f2068cdae6402a360c089a6e49a88a257
```
Comment (Dinuuu) [inline db/schema.rb]: Becareful here, you commited Conflicts into the repo

Comment (santyalcala) [inline db/schema.rb]: When i did the rebase, I must have skipped this one.
And the tests/rubocop doesn't check this file,  right?

**Pull request merged**